scripts/gopath.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I would prefer to see this setup and build harness moved into the spec if this code isn't reused elsewhere (in this PR it isn't, but maybe you have another use case on the way).

To my eyes, this external file causes things to be a bit more complicated than need be and in effect obfuscates the simple mechanism at heart of it: setting up the directory for build. The spec file can gain the necessary environment variables, placement of the project (for GOPATH building), and the build & pluck the built binaries out of the locations the project was already building them into during `%install`.

Reply:
Done